src/main/java/frc/robot/subsystems/ArmSubsystem.java

```diff
@@ -222,15 +251,26 @@ public double getChooChooAngle() {
   }
 
   public void updateReferences(double bJoystickValue, double eJoystickValue) {
-    setBaseReference(MathUtil.clamp(basePIDController.getGoal().position+Units.degreesToRadians(bJoystickValue),ArmSubsystem.kMinBaseAngle,ArmSubsystem.kMaxBaseAngle));
-    setElbowReference(MathUtil.clamp(elbowPIDController.getGoal().position+Units.degreesToRadians(eJoystickValue),ArmSubsystem.kMinElbowAngle,ArmSubsystem.kMaxElbowAngle));
+    basePIDController.setGoal(MathUtil.clamp(basePIDController.getGoal().position+Units.degreesToRadians(bJoystickValue),ArmSubsystem.kMinBaseAngle,ArmSubsystem.kMaxBaseAngle));
+    elbowPIDController.setGoal(MathUtil.clamp(elbowPIDController.getGoal().position+Units.degreesToRadians(eJoystickValue),ArmSubsystem.kMinElbowAngle,ArmSubsystem.kMaxElbowAngle));
```

Comment:
Why was this changed?

Reply:
Is this for the profiled pid controller?

Just looks like it was missed before